Doc/library/signal.rst

```diff
@@ -197,6 +197,16 @@ The variables defined in the :mod:`signal` module are:
 
    Segmentation fault: invalid memory reference.
 
+.. data:: SIGSTKFLT
+
+    Stack fault on coprocessor. The Linux kernel does not raise this signal: it
+    can only be raised in user space.
+
+   .. availability:: Linux, on platforms where the signal is available. See the
+      man page :manpage:`signal(7)` for further information.
```

Comment:
```suggestion
      manual page :manpage:`signal(7)` for further information.
```

Reply:
I didn't adopt this suggestion, because the signals documentation has 13 other occurrences of "man page" and no occurrences of "manual page", so I think it is better to be consistent with what is already there.

Reply:
Ok, I'm fine with both wordings :-)